server/xml.server.php

```diff
@@ -34,7 +34,7 @@
 
 /* Set the correct headers */
 header("Content-type: text/xml; charset=" . AmpConfig::get('site_charset'));
-header("Content-Disposition: attachment; filename=information.xml");
+// header("Content-Disposition: attachment; filename=information.xml");
```

Comment:
I agree this header is really painful to debug. But removing it might break compatibility I think. I do not know enough the XML API to have a clear idea on this. (cc @Afterster)